src/ui/Facet/ValueElement.ts

```diff
@@ -45,6 +45,7 @@ export class ValueElement {
       this.handleEventForExcludedValueElement(eventBindings);
     } else {
       this.handleEventForValueElement(eventBindings);
+      $();
```

Comment:
Yeah was trying to figure out why it wasn't failing on compilation and forgot :(